CONTRIBUTING.md

```diff
@@ -245,3 +246,60 @@ However, these options can lengthen your testing cycle and make the process of c
 ```{bash}
 ./test.sh
 ```
+
+## Releasing to CRAN <a name="cran"></a>
+
+Once substantial time has passed or significant changes have been made to `pkgnet`, a new release should be pushed to [CRAN](https://cran.r-project.org). This is a manual proces, with the following steps.
+
+### Open a PR 
+
+Open a PR with a branch name `release/v0.0.0` (replacing 0.0.0 with the actual version number).
+
+Add a section for this release to `cran-comments.md`. This file holds details of CRAN's responses to our feedback.
+
+Change the `Version:` field in `DESCRIPTION` to the official version you want on CRAN (should not have a trailing `.9000`).
+
```

Comment:
We should make sure roxygen is rebuilt too right?

Reply:
oh yep good call